src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml

```diff
@@ -41,7 +41,7 @@
                   Theme="{StaticResource FluentCaptionButton}"
                   IsVisible="False">
             <Viewbox Width="11" Margin="2">
-              <Path Name="FullScreenButtonPath"
+              <Path Name="PART_FullScreenButtonPath"
```

Comment:
This isn't actually a template part which is defined to be a control used in code behind.

Reply:
Ohh, that makes sense. I will submit another commit to revert it.

Reply:
Thanks! Everything looks good.